src/poetry/core/masonry/builders/builder.py

```diff
@@ -311,7 +311,7 @@ def convert_entry_points(self) -> dict[str, list[str]]:
                     f"Use of callable in script specification ({name}) is"
```

Comment:
I was thinking we could remove this deprecated feature and make this method a bit cleaner. It has been here for a long time now.

Reply:
I'd probably play it safe and keep it for now. (I feel like we have to be extra cautious in poetry-core (compared to poetry) because it's the backend. Remember breaking isort pre-commit hooks? 😅) Further, I'm planning to focus on PEP 621 support after the next release. Even if we try to make the transition as smooth as possible, I fear we might introduce some unintended breaking changes. Long story short, I think the version after the next release might be a major version update (if I can manage PEP 621 support) and that will be a good opportunity to remove most of the deprecated stuff.

Reply:
Ok, fine